Makefile

```diff
@@ -81,13 +81,15 @@ test-unit:
 
 .PHONY: test-integration
  test-integration:
-	@if [[ -f $(PWD)/$(CONTROL_KUBECONFIG) ]]; then export CONTROL_KUBECONFIG=$(PWD)/$(CONTROL_KUBECONFIG); fi; \
-	if [[ -f $(PWD)/$(TARGET_KUBECONFIG) ]]; then export TARGET_KUBECONFIG=$(PWD)/$(TARGET_KUBECONFIG); fi; \
+	@if [[ -f $(PWD)/$(CONTROL_KUBECONFIG) ]]; then export CONTROL_KUBECONFIG=$(PWD)/$(CONTROL_KUBECONFIG); else export CONTROL_KUBECONFIG=$(CONTROL_KUBECONFIG); fi; \
```

Comment:
this can lead to exporting even when `CONTROL_KUBECONFIG` is empty (lets say) or it doesn't point to an actual file.
a better way should be to 
```
if pwd/control_kubeconfig exists -> export
else if control_kubeconfig exists -> export
else -> print warning that no file exists on the path
```

similar could be done for the rest of variables which have this check before exporting ... 
you can skip printing warning for variables like `MACHINECLASS_V*` as they are not required in some cases.. but kubeconfigs need such warning